### PR TITLE
fix status

### PR DESCRIPTION
**Pull request by Jim8y**
this pr fix the status of nep28 and add it to readme.

### CODE REVIEW

File: README.mediawiki [hunk 1]
```diff
@@ -123,6 +123,12 @@ First review [[nep-1.mediawiki|NEP-1]]. Then clone the repository and add your N
 | Standard
 | Final
 |-
+| [[nep-28.mediawiki|28]]
+| CStandardization of the Smallest Unit of GAS
```
Comment (roman-khimov) [inline README.mediawiki:127]:
```suggestion
| Standardization of the Smallest Unit of GAS
```

**Review by roman-khimov — CHANGES_REQUESTED** [review]

**Review by shargon — DISMISSED** [review]

**Review by roman-khimov — DISMISSED** [review]

**Review by AnnaShaleva — APPROVED** [review]

File: nep-28.mediawiki [hunk 2]
```diff
@@ -2,7 +2,7 @@ NEP: 28
 Title: Standardization of the Smallest Unit of GAS
 Author: Jimmy Liao <jimmy@r3e.network>
 Type: Standard
-Status: Draft
+Status: Final
```
Comment (AnnaShaleva) [inline nep-28.mediawiki:5]: BTW, while we're here, let's fix the formatting. Standard header should be sorrounded by `<pre>` tag with corresponding indentation for proper MD rendering. Here's how does it looks like right now:
image
And here's how should it be:
image

**Review by vncoelho — APPROVED** [review]

File: nep-28.mediawiki [hunk 3]
```diff
@@ -1,9 +1,11 @@
-NEP: 28
-Title: Standardization of the Smallest Unit of GAS
-Author: Jimmy Liao <jimmy@r3e.network>
-Type: Standard
-Status: Draft
-Created: 2024-05-20
+<pre>
+    NEP: 28
+    Title: Standardization of the Smallest Unit of GAS
+    Author: Jimmy Liao <jimmy@r3e.network>
```
Comment (shargon) [inline nep-28.mediawiki:4]: If it is HTML, this should be escaped?

Comment (roman-khimov) [reply nep-28.mediawiki:4]: All the others are formatted like this and it seems to be OK: https://github.com/neo-project/proposals/blob/master/nep-16.mediawiki

**Review by roman-khimov — APPROVED** [review]

**Review by AnnaShaleva — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Pull request merged**